scripts/mikeymike.coffee

```diff
@@ -0,0 +1,76 @@
+##################################
+#
+# NOBODY READ THIS OR ELSE YOU'LL
+# RUIN THE FUN
+#
+#################################
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+module.exports = (robot) ->
```

Comment:
Trying to break my build homie. 

SyntaxError: missing ", starting
  at exports.throwSyntaxError (/Users/lydonbergin/robort/node_modules/hubot/node_modules/coffee-script/lib/coffee-script/helpers.js:197:13)
  at Lexer.exports.Lexer.Lexer.error (/Users/lydonbergin/robort/node_modules/hubot/node_modules/coffee-script/lib/coffee-script/lexer.js:778:14)
  at Lexer.exports.Lexer.Lexer.balancedString (/Users/lydonbergin/robort/node_modules/hubot/node_modules/coffee-script/lib/coffee-script/lexer.js:585:19)
  at Lexer.exports.Lexer.Lexer.stringToken (/Users/lydonbergin/robort/node_modules/hubot/node_modules/coffee-script/lib/coffee-script/lexer.js:178:31)
  at Lexer.exports.Lexer.Lexer.tokenize (/Users/lydonbergin/robort/node_modules/hubot/node_modules/coffee-script/lib/coffee-script/lexer.js:30:143)
  at exports.compile.compile (/Users/lydonbergin/robort/node_modules/hubot/node_modules/coffee-script/lib/coffee-script/coffee-script.js:35:36)
  at Object.loadFile (/Users/lydonbergin/robort/node_modules/hubot/node_modules/coffee-script/lib/coffee-script/coffee-script.js:176:14)
  at Module.load (/Users/lydonbergin/robort/node_modules/hubot/node_modules/coffee-script/lib/coffee-script/coffee-script.js:211:36)
  at Function.Module._load (module.js:312:12)
  at Module.require (module.js:364:17)
  at require (module.js:380:17)
  at Robot.loadFile (/Users/lydonbergin/robort/node_modules/hubot/src/robot.coffee:217:9, <js>:163:20)
  at Robot.load (/Users/lydonbergin/robort/node_modules/hubot/src/robot.coffee:239:9, <js>:186:30)
  at Shell.loadScripts (/Users/lydonbergin/robort/node_modules/hubot/bin/hubot:93:5, <js>:94:13)
  at Shell.emit (events.js:92:17)
  at Object.Readline.createInterface.next (/Users/lydonbergin/robort/node_modules/hubot/src/adapters/shell.coffee:59:9, <js>:104:17)
  at /Users/lydonbergin/robort/node_modules/hubot/node_modules/readline-history/readline-history.js:112:21
  at ReadStream.onend (/Users/lydonbergin/robort/node_modules/hubot/node_modules/readline-history/readline-history.js:52:21)
  at ReadStream.emit (events.js:117:20)
  at _stream_readable.js:944:16
  at process._tickCallback (node.js:442:13)